### PR TITLE
🌐 Add Japanese translation for Features

### PR DESCRIPTION
**Pull request by tokusumi**
This PR translates the features.md to Japanese.
Please review features.md.

~Issue #1571 generates a new directory to support Japanese. 
I will fix dependency after #1571 is merged.~

### CODE REVIEW

File: docs/ja/docs/features.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+
+![editor support](https://fastapi.tiangolo.com/img/pycharm-completion.png)
+
+以前は不可能だと考えていたコードでさえ補完されます。例えば、リクエストからのJSONボディ（ネストされている可能性がある）内の `price`キーです。
+
+間違ったキー名を入力したり、ドキュメント間を行き来したり、上下にスクロールして`username`と`user_name`のどちらを使用したか調べたりする必要はもうありません。
+
+### 簡潔
+
+すべてに適切な**デフォルト**があり、どこにでもオプションの構成があります。必要なことを実行し、必要なAPIを定義するためにすべてのパラメーターを調整できます。
```
Comment (SwftAlpc) [inline docs/ja/docs/features.md:97]: すべてに適切な**デフォルト**があり、オプションの構成ができます。

File: docs/ja/docs/features.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+
+さらに、Starletteのすべてのセキュリティ機能も含みます（**セッションCookie**を含む）。
+
+これらは、システム、データストア、リレーショナルデータベース、NoSQLデータベースなどと簡単に統合できる再利用可能なツールとコンポーネントとして構築されています。
+
+### 依存性の注入（Dependency Injection）
+
+FastAPIには非常に使いやすく、非常に強力な<abbr title='also known as "components", "resources", "services", "providers"'><strong>依存性の注入</strong></abbr>システムを備えています。
+
+* 階層または **依存関係の"グラフ"** を作成することで、依存関係でさえも依存関係を持つことができます。
```
Comment (SwftAlpc) [inline docs/ja/docs/features.md:138]: 依存関係でさえも依存関係を持つことができ、階層または **依存関係の"グラフ”** を作成することができます。

File: docs/ja/docs/features.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+データベースのために<abbr title = "Object-Relational Mapper">ORM</abbr>sや、<abbr title = "Object-Document Mapper">ODM</abbr>sなどの、Pydanticに基づく外部ライブラリを備えています。
+
+これは、すべてが自動的に検証されるため、多くの場合、リクエストから取得したオブジェクトを**データベースに直接**渡すことができるということを意味しています。
+
+同じことがその逆にも当てはまり、多くの場合、データベースから取得したオブジェクトを**クライアントに直接**渡すことができます。
+
+**FastAPI**を使用すると、**Pydantic**のすべての機能を利用できます（FastAPIがPydanticに基づいてすべてのデータ処理を行っているため）。
+
+* **brainfuckなし**：
+    * 学習するべき新しいスキーマ定義のマイクロ言語はありません。
```
Comment (SwftAlpc) [inline docs/ja/docs/features.md:191]: 新しいスキーマ定義のマイクロ言語を学習する必要はありません。

**Review by SwftAlpc — CHANGES_REQUESTED** [review]
Thank you for the excellent translation.
There are a few things that I noticed and I would like to suggest some corrections.

**Comment by tokusumi** [comment]
@SwftAlpc Thank you for your good suggestions. I improved translation. Please re-review and approve this PR if you want.

**Review by SwftAlpc — APPROVED** [review]
Thanks for the quick response.
LGTM!

**Review by ryuckel — APPROVED** [review]

**Comment by tokusumi** [comment]
Hi, @tiangolo  
This PR has been approved by two reviewers (github actions to label approved PR didn't work, though). 
Would you like to require something more for this to be merged?

**Comment by tiangolo** [comment]
Awesome! Thank you @tokusumi ! :bow: :cake: 

And thank you @ryuckel and @SwftAlpc for the reviews :clap: :coffee: 

Indeed, the GitHub action to manage the labels wasn't working correctly, so I just created a new one that is finally working :smile: :robot:

**Pull request merged**